src/ResourceManager/Network/Commands.Network/Cortex/HubVnetConnection/NewAzureRmHubVirtualNetworkConnectionCommand.cs

```diff
@@ -0,0 +1,176 @@
+            HelpMessage = "Run cmdlet in the background")]
+        public SwitchParameter AsJob { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Do not ask for confirmation if you want to overrite a resource")]
+        public SwitchParameter Force { get; set; }
+
+        public override void Execute()
+        {
+            base.Execute();
+            WriteWarning("The output object type of this cmdlet will be modified in a future release.");
+
+            if (ParameterSetName.Equals(CortexParameterSetNames.ByVirtualHubObject, StringComparison.OrdinalIgnoreCase))
+            {
+                this.ResourceGroupName = this.ParentResource.ResourceGroupName;
+                this.ParentResourceName = this.ParentResource.Name;
+            }
+            else if (ParameterSetName.Equals(CortexParameterSetNames.ByVirtualHubResourceId, StringComparison.OrdinalIgnoreCase))
+            {
+                var parsedResourceId = new ResourceIdentifier(this.ParentResourceId);
+                this.ResourceGroupName = parsedResourceId.ResourceGroupName;
+                this.ParentResourceName = parsedResourceId.ResourceName;
+            }
+
+            //// Get the virtual hub - this will throw not found if the resource does not exist
+            PSVirtualHub parentVirtualHub = this.GetVirtualHub(this.ResourceGroupName, this.ParentResourceName);
+            if (parentVirtualHub == null)
+            {
+                throw new PSArgumentException("The parent virtual hub mentioned could not be found.");
+            }
+
+            PSHubVirtualNetworkConnection hubVnetConnection = new PSHubVirtualNetworkConnection();
+            hubVnetConnection.Name = this.Name;
+            hubVnetConnection.EnableInternetSecurity = this.EnableInternetSecurity.HasValue ? this.EnableInternetSecurity.Value : false;
+
+            //// Resolve the remote virtual network
+            //// Let's not try to resolve this since this can be in other RG/Sub/Location
+            if (this.RemoteVirtualNetwork != null)
+            {
+                hubVnetConnection.RemoteVirtualNetwork = new PSResourceId() { Id = this.RemoteVirtualNetwork.Id };
+            }
+            else if (!string.IsNullOrWhiteSpace(this.RemoteVirtualNetworkId))
+            {
+                hubVnetConnection.RemoteVirtualNetwork = new PSResourceId() { Id = this.RemoteVirtualNetworkId };
+            }
+            else
+            {
+                throw new PSArgumentException("A remote virtual network reference is required to create a HubVirtualNetworkConnection.");
+            }
+
+            if (parentVirtualHub.VirtualNetworkConnections == null)
+            {
+                parentVirtualHub.VirtualNetworkConnections = new List<PSHubVirtualNetworkConnection>();
+            }
+
+            parentVirtualHub.VirtualNetworkConnections.Add(hubVnetConnection);
+
+            bool shouldProcess = this.Force.IsPresent;
```

Comment:
Force is only used for actions that are very destructive.  Please only use them on Removes, and for Remove, only if that remove is a particularly destructive actions (aka, if it destroys child resources or is a resource that is very expensive and hard to create)

Reply:
Removed Force from all New/Set commands.